src/features/enrollments/EnrollmentStatCard.jsx

```diff
@@ -19,7 +20,7 @@ export default function EnrollmentStatsCard({ partner, onlyComplete }) {
   }).length;
   const unit = enrollmentTotal === 1 ? singleUnit : pluralUnit;
 
-  return <StatCard value={enrollmentTotal} unit={unit} />;
+  return <StatCard value={enrollmentTotal} unit={unitText} secondary={unit} />;
```

Comment:
I think we could avoid the assignment on line 8 and just return this...

```suggestion
  return <StatCard value={enrollmentTotal} unit="Total Course" secondary={unit} />;
```